sorc/link_workflow.sh

```diff
@@ -245,6 +245,21 @@ if [[ -d "${HOMEgfs}/sorc/gdas.cd" ]]; then
   done
 fi
 
+#------------------------------
+#--add SPOC parm and ush directory
+#------------------------------
+sources=("config" "scripts")
+targets=("parm" "ush")
```

Comment:
@xincjin-NOAA instead of linking this into `parm`, could we link this into `parm/gdas` instead?  The latter already exists and works as a base for all GDASApp parm files.

```suggestion
targets=("parm/gdas" "ush")
```

Reply:
changed.
Thanks